x-pack/plugins/security_solution/public/entity_analytics/components/entity_store/components/dashboard_panels.tsx

```diff
@@ -0,0 +1,246 @@
+ * 2.0.
+ */
+import React, { useState } from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiToolTip,
+  EuiButton,
+  EuiLoadingSpinner,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiLoadingLogo,
+  EuiPanel,
+  EuiImage,
+} from '@elastic/eui';
+
+import { FormattedMessage } from '@kbn/i18n-react';
+import { RiskEngineStatusEnum } from '../../../../../common/api/entity_analytics';
+import { RiskScoreEntity } from '../../../../../common/search_strategy';
+
+import { EntitiesList } from '../entities_list';
+
+import { useEntityStoreEnablement } from '../hooks/use_entity_store';
+import { EntityStoreEnablementModal, type Enablements } from './enablement_modal';
+
+import { EntityAnalyticsRiskScores } from '../../entity_analytics_risk_score';
+import { useInitRiskEngineMutation } from '../../../api/hooks/use_init_risk_engine_mutation';
+import { useEntityEngineStatus } from '../hooks/use_entity_engine_status';
+
+import dashboardEnableImg from '../../../images/entity_store_dashboard.png';
+import {
+  ENABLEMENT_INITIALIZING_ENTITY_STORE,
+  ENABLEMENT_INITIALIZING_RISK_ENGINE,
+  ENABLE_ALL_TITLE,
+  ENABLE_ENTITY_STORE_TITLE,
+  ENABLE_RISK_SCORE_TITLE,
+} from '../translations';
+import { useRiskEngineStatus } from '../../../api/hooks/use_risk_engine_status';
+
+const EntityStoreDashboardPanelsComponent = () => {
+  const [modal, setModalState] = useState({ visible: false });
+  const [riskEngineInitializing, setRiskEngineInitializing] = useState(false);
+
+  const entityStore = useEntityEngineStatus();
+  const riskEngineStatus = useRiskEngineStatus();
+
+  const { enable: enableStore } = useEntityStoreEnablement();
+  const { mutate: initRiskEngine } = useInitRiskEngineMutation();
+
+  const enableEntityStore = (enable: Enablements) => () => {
+    setModalState({ visible: false });
+    if (enable.riskScore) {
+      const options = {
+        onSuccess: () => {
+          setRiskEngineInitializing(false);
+          if (enable.entityStore) {
+            enableStore();
+          }
+        },
+      };
+      setRiskEngineInitializing(true);
+      initRiskEngine(undefined, options);
+    }
+
+    if (enable.entityStore) {
+      enableStore();
+    }
+  };
+
+  if (entityStore.status === 'loading') {
+    return (
+      <EuiPanel hasBorder>
+        <EuiEmptyPrompt
+          icon={<EuiLoadingSpinner size="xl" />}
+          title={<h2>{ENABLEMENT_INITIALIZING_ENTITY_STORE}</h2>}
+        />
+      </EuiPanel>
+    );
+  }
+
+  if (entityStore.status === 'installing') {
+    return (
+      <EuiPanel hasBorder>
+        <EuiEmptyPrompt
+          icon={<EuiLoadingLogo logo="logoElastic" size="xl" />}
+          title={<h2>{ENABLEMENT_INITIALIZING_ENTITY_STORE}</h2>}
+          body={
+            <p>
+              <FormattedMessage
+                id="xpack.securitySolution.entityAnalytics.entityStore.enablement.initializing.description"
+                defaultMessage="It can take up to 5 minutes"
```

Comment:
What does "It" refer to?

Reply:
The initializing process. 

![Screenshot 2024-10-15 at 14 11 32](https://github.com/user-attachments/assets/c94c5a29-4a99-4273-88cc-17a77a301dd4)

Reply:
Thanks for the screenshot! Just a minor suggestion:

```suggestion
                defaultMessage="This can take up to 5 minutes."
```